adhocracy4/ratings/static/ratings/react_ratings.jsx

```diff
@@ -23,8 +25,10 @@ var RatingBox = React.createClass({
   },
   handleRatingModify: function (number, id) {
     api.rating.change({
-      object_pk: this.props.objectId,
-      content_type: this.props.contentType,
+      urlReplaces: {
```

Comment:
Why do the components suddenly require knowledge about the url. I thougt the `api.rating` and `api.comments` where meant to abstract that away.